src/episerver_fmt_plug.c

```diff
@@ -298,7 +298,7 @@ static void set_salt(void *salt)
 #ifdef SIMD_COEF_32
 	int index, j;
 	cur_salt = (struct custom_salt *)salt;
-	for (index = 0; index < MAX_KEYS_PER_CRYPT*omp_t; ++index)
+	for (index = 0; index < MAX_KEYS_PER_CRYPT*threads; ++index)
```

Comment:
This is probably a bug - should probably use actual max_keys_per_crypt here, which may be higher by a factor of OMP_SCALE. Better yet, this logic should be in crypt_all(), so it'd go up to the actual count rather than up to max.